### ISSUE TITLE
Seed for validation

### ISSUE THREAD

**Issue by amartya18x** [gpleiss/temperature_scaling#14]
Hi,

I was wondering if you could share the seed for pytorch random. We wanted to use the same seed to get the same validation set as yours. 

Thanks
